app/views/jobapps_mailer/saved_applications_notification.text.erb

```diff
@@ -0,0 +1,10 @@
+Hello,
+
+The following applications have been reactivated:
+<% @info.each_key do |position| %>
+  Position: <%= position %>
+  <% @info[position].each do |record| %>
+    Applicant: <%= record.user.full_name %>,
+    Date Applied: <%= record.created_at.strftime('%m/%d/%Y %H:%M') %>
```

Comment:
Given that the value is a date, I think just `Applied:` or `Applied on:` will be perfectly clear, though if you disagree I don't have strong feelings about this.